scripts/manifests/chromemanifest.json

```diff
@@ -40,7 +40,7 @@
 
   "permissions": [
     "storage",
-    "activeTab",
+    "<all_urls>",
```

Comment:
Note we anyway do this for all context scripts, so the permission that the browser asks the user for should be the same. This here is required, because `menu.onShown` otherwise does not get information about the selected text.